src/TestInfrastructure/TestInfrastructure.ruleset

```diff
@@ -5,6 +5,7 @@
     <Rule Id="CS1072" Action="None" />
   </Rules>
   <Rules AnalyzerId="SonarLint.CSharp" RuleNamespace="SonarLint.CSharp">
+    <Rule Id="S107" Action="None" />
```

Comment:
lambda has more than X - in our cases its not a lambda it's an explicit implementation of VS interfaces (and we have quite a few of them in this test project).